tools/gstswitchserver.c

```diff
@@ -1519,14 +1519,7 @@ gst_switch_server_start_output (GstWorker * worker, GstSwitchServer * srv)
 static void
 gst_switch_server_start_recorder (GstWorker * worker, GstSwitchServer * srv)
 {
-  g_return_if_fail (GST_IS_WORKER (worker));
-
-  GST_SWITCH_SERVER_LOCK_CONTROLLER (srv);
-  if (srv->controller) {
-    gst_switch_controller_tell_encode_port (srv->controller,
-        srv->composite->encode_sink_port);
-  }
-  GST_SWITCH_SERVER_UNLOCK_CONTROLLER (srv);
+  INFO ("encoding-output started");
```

Comment:
The clients usually request them using an active Call: `get_encode_port`. The Signal raised by `gst_switch_controller_tell_encode_port` was not used anyway, because it would require a Client to be connected *before* the Server opens its encode-Port -- which is practically impossible.

As you can see the Integrationtests (ie TestGetEncodePort.get_encode_port) still work ;)